test/common/fixtures.js

```diff
@@ -14,6 +14,10 @@ function fixturesFileURL(...args) {
   return pathToFileURL(fixturesPath(...args));
 }
 
+function fixturesFileURLString(...args) {
+  return JSON.stringify(fixturesFileURL(...args));
```

Comment:
```suggestion
function fixturesFileURLString(...args) {
  return JSON.stringify(ReflectApply(fixturesFileURL, null, args)));
```
so as to not depend on `Array.prototype[Symbol.iterator]`